### DISCUSSION TITLE
i need to capture the footer section of the page only

### DISCUSSION THREAD

**Discussion by brunofelix1711** [percy/percy-selenium-java#86] [Q&A]
Hi All, 

I am using selenium java BDD framework , I need to capture the specific region or section of the page like footer section but when I use the below code , I did not get the screen shot in Percy dashboard  , kindly advise .

```java
driver.get("https://example.com");
percy.snapshot('name', null, 300, true, "classname");
```

**Comment by Robdel12** [comment]
Hey @brunofelix1711! Percy can't really capture specific parts of the page. There is a way to do this, though. You can get creative and use Percy CSS to hide all the elements on the page that aren't what you want to capture. How this is done is very dependent on your pages CSS, though, so I can't give any code that would work. The general idea is something like:


```java
driver.get("https://example.com");

percy.snapshot('Footer', null, 300, false, "* {visibility: none; } .footer-selector { visibility: visible !important; }");
```

**Reply by brunofelix1711** [reply]
HI Robdel ,
Thanks for help and i have a below doubt , much appreciated if clarify this thing too :) 
suppose if i need the footer section only and hide rest of items in the page , i have the class name of footer like this footer-xxx, so i update like a below, is it correct ? and also i need to make false the java script executor ? 

percy.snapshot('Footer', null, 300, false, "* {visibility: none; } .footer-xxx { visibility: visible !important; }");
